training-maquetado/index.html

```diff
@@ -10,18 +10,102 @@
     <link rel="stylesheet" type="text/css" href="application.css">
   </head>
   <body>
-  <div class="container">
-    <h2 class="title text-1">Front-End Tools</h2>
-    <div class="tech-container">
-      <div class="tech-card">
-        <img class="tech-logo" src="assets/react_icon.svg" alt="">
-        <p class="tech-name text-2">ReactJS</p>
-        <p class="tech-release text-3">Release: 2013</p>
-        <p class="tech-description">ReactJS is a declarative component-based JavaScript library for building user
-          interfaces.</p>
+    <section class="section-container">
+      <header>
+        <h2 class="title text-1">Front-End Tools</h2>
+      </header>
```

Comment:
why are you wrapping this in a header?

Reply:
Only because w3school recommend it https://www.w3schools.com/html/html5_semantic_elements.asp